docs/volunteering/index.md

```diff
@@ -0,0 +1,33 @@
+# Getting Involved
+
+Student Robotics is powered by volunteers. We have a variety of skills and
+backgrounds, but share a passion for encouraging the teams through the medium of
+robotics.
+
+There are a lots of different ways you can get involved. Whether it's mentoring
+a team through the competition year, helping run one of our events or developing
+our hardware or software, this runbook is the place to look for guidance on how
+we do things.
+
+## What to expect
+
+When you first start helping out, you'll have a chat with a volunteer
+co-ordinator who will be able to tell you a bit more about how we're organised
+and work with you to find things you'd like to help with. They'll also ask you
+for contact details and for you to read the [Code of Conduct][code-of-conduct].
+
+They'll also be able to answer any questions you have about the organisation and
+get you set up with accounts on our collaboration platforms (we primarily use
+[Slack][sr-slack] and [GitHub][sr-github]).
+
+[code-of-conduct]: https://srobo.gitbook.io/ops-manual/about-the-charity/code-of-conduct
+[sr-slack]: https://studentrobotics.slack.com/
+[sr-github]: https://github.com/srobo
+
```

Comment:
I'm probably going to introduce a "What we expect from you" section here, but I'm not sure if that'll make it into this PR. I think we need such a section, but clearly the wording needs to be both clear and empathetic and therefore needs care in its crafting.

Reply:
"What we expect from you" is definitely not the right wording. How about "Volunteer responsibilities"?

Reply:
c18737cec636f55457ed2fb0942d444122a1dee1 adds such a section.